### PR TITLE
[NUI] Deprecate set properties of Color, Rectangle, and etc

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Deprecate set properties of Color, Rectangle, and etc
- https://code.sec.samsung.net/jira/browse/TCSACR-376
- The deprecated setters will be changed as internal APIs when Remove-ACR.

### API Changes ###
https://code.sec.samsung.net/jira/browse/TCSACR-376

### CODE REVIEW

**Review by rabbitfor — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/Position.cs [hunk 1]
```diff
@@ -521,10 +521,13 @@ public static Position Zero
         /// The x component.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor")]
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:524]:
```suggestion
            [Obsolete("Do not use this setter, which is deprecated in API8 and will be removed in API10. Use the new Position(...) constructor")]
```

File: src/Tizen.NUI/src/public/Position.cs [hunk 2]
```diff
@@ -521,10 +521,13 @@ public static Position Zero
...
         public float X
         {
             set
             {
+                Tizen.Log.Fatal("NUI", "Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor");
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:529]:
```suggestion
            [Obsolete("Do not use this setter, which is deprecated in API8 and will be removed in API10. Use the new Position(...) constructor")]
```

File: src/Tizen.NUI/src/public/Position.cs [hunk 3]
```diff
@@ -542,10 +545,13 @@ public float X
         /// The y component.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor")]
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:548]:
```suggestion
        [Obsolete("Do not use this setter, which is deprecated in API8 and will be removed in API10. Use the new Position(...) constructor")]
```

File: src/Tizen.NUI/src/public/Position.cs [hunk 4]
```diff
@@ -542,10 +545,13 @@ public float X
...
         public float Y
         {
             set
             {
+                Tizen.Log.Fatal("NUI", "Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor");
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:553]:
```suggestion
                Tizen.Log.Fatal("NUI", "Please do not use this setter, which is deprecated in API8 and will be removed in API10. Please use the new Position(...) constructor.");
```

File: src/Tizen.NUI/src/public/Position.cs [hunk 5]
```diff
@@ -563,10 +569,13 @@ public float Y
         /// The z component.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor")]
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:572]:
```suggestion
        [Obsolete("Do not use this setter, which is deprecated in API8 and will be removed in API10. Use the new Position(...) constructor".)]
```

File: src/Tizen.NUI/src/public/Position.cs [hunk 6]
```diff
@@ -563,10 +569,13 @@ public float Y
...
         public float Z
         {
             set
             {
+                Tizen.Log.Fatal("NUI", "Please do not use this setter, Deprecated in API8, will be removed in API10. please use new Position(...) constructor");
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Position.cs:577]:
```suggestion
                Tizen.Log.Fatal("NUI", "Please do not use this setter, which is deprecated in API8 and will be removed in API10. Please use the new Position(...) constructor.");
```

**Review by journey2w — COMMENTED** [review]
I didn't write to give you an update on all descriptions.
Please update the same sentences,

File: src/Tizen.NUI/src/public/Position.cs (hunk 1 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:524]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

File: src/Tizen.NUI/src/public/Position.cs (hunk 3 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:548]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

File: src/Tizen.NUI/src/public/Position.cs (hunk 2 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:529]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

File: src/Tizen.NUI/src/public/Position.cs (hunk 4 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:553]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

File: src/Tizen.NUI/src/public/Position.cs (hunk 5 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:572]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

File: src/Tizen.NUI/src/public/Position.cs (hunk 6 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Position.cs:577]: Thank you for review.
these are fixed by the following patches.

https://github.com/Samsung/TizenFX/pull/4356
https://github.com/Samsung/TizenFX/pull/4357

thank you.

**Pull request merged**